### ISSUE TITLE
add an option to check all paths of the title when an install action is taken.

### ISSUE THREAD

**Issue by CuddleBear92** [game-list/game-list#41]
Add an option to check all paths of the title when an install action is taken.

This way if you install a new update it should check all the paths of the other versions, giving the user the option to have older un-installed versions cleaned out at the same time.
This would also double as a version mapping clean-up of the newly deleted version (when delete on install is used)

Thoughts are that it cleans up all mappings that is no longer on the system for whatever reason.
So you don't have versions mapped and marked as installed that you might not have files for.

This should all be an toggleable option as users with external drives and such would want control over it.

In short:
- Check all paths of all versions at the end of install workflow.
- Remove mappings/metadata of missing versions.
- Make it an on/off option to do this on installs.
